### ISSUE TITLE
Country profiles

### ISSUE THREAD

**Issue by slaz0** [thewca/worldcubeassociation.org#1291]
Sorry if this has already been suggested but individual countries and continents should have their own profiles. They could show national/continental records in every event and how they compare to the rest of the world. The profiles could show the history of their national/continental records, the people that rank highest when comparing their best singles/averages to the rest of the country, the number of competitions held and similar statistics. Please do comment any problems this might cause!

**Comment by jfly** [comment]
This reminds me of this cool page from Daniel Sheppard: http://www.kinch2002.com/kinchranks/countries/

**Comment by slaz0** [comment]
Yes, I do love kinchranks :)

Also, I am aware that there are country summaries on the WCA, however, these only show the best singles/averages and history. The country profiles could show flags and location on a world map as well as the statistics I mentioned above in addition to what can already be found on the WCA.

**Comment by jfly** [comment]
I just learned that @oyyq99999 maintains an up to date country kinchranks list at http://wca.cuber.pro/.

I'll ping Daniel Sheppard about updating http://www.kinch2002.com/kinchranks/ to include a link to it.

**Comment by oyyq99999** [comment]
Yeah that was set up since like one year ago. I didn't announce it because it was not finished (as I expected). Recently I'm very busy on working and practicing cubing. I'm planning to merge the code of both sites (_wca.cuber.pro_ and _algs.cuber.pro_) and make _algs.cuber.pro_ more useful. But I don't know when. Hopefully in April.

**Comment by ronaldmansveld** [comment]
Also, up to date kinchranks are available on wcadb.net/org (forgot which
TLD they use)
